yarn.lock

```diff
@@ -17553,11 +17553,16 @@ karma@^5.0.4:
     ua-parser-js "0.7.21"
     yargs "^15.3.1"
 
-keyborg@^0.7.0, keyborg@^0.7.0-alpha.1:
+keyborg@^0.7.0-alpha.1:
```

Comment:
As we discussed in offline, we should a single version of `keybord`, so we need to bump it in `packages/react-tabster/package.json`.

Reply:
can you please provide more context for non offline folks ? 🙏

Reply:
This package used to be a part of `tabster` but was moved out to a separate package so that consuming it would not blow up bundle size

Reply:
so before it was direct dependency of tabster, now it's out. that's why bumping both tabster and keyborg to get only 1 dependency version within `yarn.lock` ?